migtests/scripts/functions.sh

```diff
@@ -432,6 +442,12 @@ import_data_to_source_replica() {
 	then
 		args="${args} --source-replica-db-schema ${SOURCE_REPLICA_DB_SCHEMA}"
 	fi
+	if [ "${SOURCE_DB_ORACLE_CDB_TNS_ALIAS}" != "" ]
```

Comment:
this should jsut be `SOURCE_DB_ORACLE_TNS_ALIAS` not with `CDB`?

Reply:
For live migration `SOURCE_DB_ORACLE_CDB_TNS_ALIAS` is required so setting that and not `SOURCE_DB_ORACLE_TNS_ALIAS`. It was getting a bit tricky to run all types of combinations within the same test. Thus the introduction of live_env.sh.

Reply:
env var `SOURCE_REPLICA_DB_ORACLE_TNS_ALIAS=ORCLPDB1SSL`